plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -331,11 +341,36 @@ func (e *quotaEvaluator) checkRequest(quotas []api.ResourceQuota, a admission.At
 		return quotas, nil
 	}
 
+	// if we have limited resources enabled for this resource, always calculate usage
+	inputObject := a.GetObject()
+	mustHaveCoveringQuota := []api.ResourceName{}
+	for _, limitedResource := range e.config.LimitedResources {
+		limitedGroupResource := schema.GroupResource{Group: limitedResource.APIGroup, Resource: limitedResource.APIResource}
```

Comment:
was the concern the cost of regex evaluation or the cost of evaluating usage?

for all but one use case in openshift, per object usage calculation is cheap.  i guess if regex matching was a problem, we could keep a cache on compiled expressions in future.